### ISSUE TITLE
support aria-valuenow on other roles which might represent values?

### ISSUE THREAD

**Issue by brennanyoung** [w3c/aria#1221]
In a concrete use case, we have a series of buttons which record how many times they have been clicked. The tally is displayed on the button beside the label. There is no way to count down or reset the tally. In a sense, the tally is read-only, in that it can only be modified via clicks, rather than by text entry.

This behavior falls somewhere between a push button and a spinbutton. 

I should add that we have dozens of these buttons, so a clean and lean presentation is preferred.,

If I mark up the control as a `spinbutton`, I believe I am required to support all the other spin button behavior, such as reset and decrement. The `spinbutton` role will raise certain expectations about use which we simply wont fulfil.

If I mark up the control as a `button`, then the `aria-valuenow` (and related aria value attributes) will not be supported. (i.e. it is likely to fail). This is how I interpret [the ARIA spec on attribute processing](https://www.w3.org/TR/wai-aria/#state_property_processing) at least.

Isn't it reasonable that widgets other than 

- `range`
- `scrollbar`
- `separator`
- `slider`
- `spinbutton`
- `progressbar`

... might have a value associated with them? How else will I communicate the tally in an accessible way? Should I piggyback on the `aria-label`? That seems kinda hacky, especially when there is an aria property intended for communicating this kind of thing in other roles.

Just as a button becomes a menu button or a toggle button not by changing its role, but by adding attributes, could we not conceive other common button subtypes, whose semantics is determined by existing optional attributes? 

A 'recent posts' button with a notification tally is a similar and very common case, so I don't think this case is all that exotic. 

If this has been discussed somewhere, please direct me to that thread, and this issue can be closed.

**Comment by brennanyoung** [comment]
Ha! I notice @smhigley just asked a very similar question at https://github.com/w3c/aria/issues/1220

**Comment by carmacleod** [comment]
@brennanyoung Are you ok with closing this as a dup of  #711?

**Comment by jnurthen** [comment]
Closing - please reopen if you don't agree.
